src/applications/vaos/referral-appointments/flow.js

```diff
@@ -51,6 +53,10 @@ export function routeToPageInFlow(history, current, action, referralId) {
   const nextPageString = pageFlow[current][action];
   const nextPage = pageFlow[nextPageString];
 
+  if (action === 'next' && nextPageString === 'scheduleReferral') {
+    startReferralTimer(referralId);
+  }
+
```

Comment:
This seemed like a good place to trigger the start timer but I can also see moving it to the 2 places that trigger the flow explicitly and out of this function

Reply:
I like it here, makes it simple and I like having it in one place.